packages/esm-patient-registration-app/src/patient-registration/field/person-attributes/coded-person-attribute-field.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './../field.scss';
+import { Input } from '../../input/basic-input/input/input.component';
+import { Select, SelectItem } from 'carbon-components-react';
+import { useConceptAnswers } from './person-attributes.resource';
+import { PersonAttributeTypeResponse } from '../../patient-registration-types';
+
+export interface CodedPersonAttributeFieldProps {
+  personAttributeType: PersonAttributeTypeResponse;
+  answerConceptSetUuid: string;
+  label?: string;
+}
+
+export function CodedPersonAttributeField({
+  personAttributeType,
+  answerConceptSetUuid,
+  label,
+}: CodedPersonAttributeFieldProps) {
+  const { data: conceptAnswers, isLoading: isLoadingConceptAnswers } = useConceptAnswers(answerConceptSetUuid);
+
+  return (
+    <div className={`${styles.attributeField} ${styles.halfWidthInDesktopView}`}>
+      {!isLoadingConceptAnswers && conceptAnswers?.length ? (
+        <Select
+          id={`person-attribute-${personAttributeType.uuid}`}
+          name={`attributes.${personAttributeType.uuid}`}
+          labelText={label ?? personAttributeType?.display}
+          light>
+          {conceptAnswers.map((answer) => (
+            <SelectItem key={answer.uuid} value={answer.uuid} text={answer.display} />
+          ))}
```

Comment:
It would be a good idea to add a default blank option here. Otherwise it appears as if the first concept answer is always selected. Cf. how [Carbon-React implements placeholders](https://react.carbondesignsystem.com/?path=%2Fdocs%2Fcomponents-select--default).